frontend/components/pages/members/list-item.tsx

```diff
@@ -1,14 +1,23 @@
-import { Box } from "@chakra-ui/core";
+import { getStrapiMedia } from "lib/media";
+import Image from "next/image";
 import React from "react";
 import { IMember } from "types/member";
 
 export const MembersListItemComponent: React.FC<{
   member: IMember;
 }> = ({ member }) => (
-  <Box>
+  <div style={{ border: "0.8px solid #ddd", margin: "0 3px" }}>
     <div>
-      <div>Name{member.Name}</div>
-      <div>Email {member.Email}</div>
+      {member.Photo && <Image src={getStrapiMedia(member.Photo)} alt={`${member.Name}`} height="300px" width="300px" />}
```

Comment:
If fixing height and width it is also important to set an `objectFit` property so that images do not stretch in the wrong way. I would recommend _cover_ for the profile images
https://www.w3schools.com/css/css3_object-fit.asp

Reply:
@chrismclarke 
Meant to add that when the _bio pop-up_ was implemented. But can still add that to the style